### PR TITLE
Update symposium2.md

### PR DESCRIPTION
**Pull request by leticiaseixas**

### CODE REVIEW

**Comment by leticiaseixas** [comment]
@SteveALee do you know why this PR is not generating a preview?

**Comment by SteveALee** [comment]
@leticiaseixas  I think it's because this is not a PR to the master (default) branch which netlify is monitoring. I explicitly added this branch and it may now work if you make another commit to your branch. In fact I seem to recall it runs twice :(

**Comment by SteveALee** [comment]
I removed the w3c: prfix form the Netlify settings on the hunch that would cause problems

**Comment by SteveALee** [comment]
@leticiaseixas Yay!

**Comment by SteveALee** [comment]
@leticiaseixas Not Yay!

it failed due to dependencies again!

**Comment by leticiaseixas** [comment]
> @leticiaseixas Not Yay!
> 
> it failed due to dependencies again!

Yes, so, the same issue. Updating with the latest changes on master works, but then I have a lot of new files modified - not a great thing for merging though.

**Comment by SteveALee** [comment]
I dont' follow? You mean you merged all the changes form master into you branch. That should be OK.

Crazy

**Comment by leticiaseixas** [comment]
I modified just one file (the one related to the symposium), the other ones were merged from master into my branch.
That should be ok.

**Comment by SteveALee** [comment]
Oh yes, it's good practice to do that regularly so clashes don't build up. :)

**Comment by shawna-slh** [comment]
/me peeks in to see if it's ready to publish... sees it's still a draft. waits patiently :-)

File: _about/projects/wai-coop/symposium2.md [hunk 1]
```diff
@@ -29,109 +29,102 @@ github:
...
+
+Participation at the event is free of charge, but registration is compulsory. The registration deadline is 5 January 2023.
 
 ## Introduction
 {:#introduction}
 
-This online symposium brings together researchers, academics, industry, government, and people with disabilities, to explore the current state-of-the-art in accessibility research, development, and practice to uncover pressing topics for future research and development.
+This online symposium brings together researchers, academics, industry, government, and people with disabilities, to explore one the most pressing emerging technologies, Artificial Intelligence, and potential impacts on accessibility. This symposium aims to identify current challenges and opportunities raised by the increasing use of AI regarding digital accessibility and explore how ongoing research can leverage and hinder digital accessibility.
```
Comment (iadawn) [inline _about/projects/wai-coop/symposium2.md:42]: Would suggest dropping:

> ', and potential impacts on accessibility'

File: _about/projects/wai-coop/symposium2.md [hunk 2]
```diff
@@ -29,109 +29,102 @@ github:
...
+- Shivam Singh (Accenture, India)
+- Michael Cooper (W3C, US)
+- Shaomei Wu (AImpower.org, US)
 
--	Problem addressed
--	Relevant background
--	Challenges
--	Outcomes
--	Future perspectives
+This panel aims to discuss the current state of natural language processing techniques employed, among other, to generate textual descriptions for images and other visual media present on web pages. This discussion will be focused on aspects such as text adaptation, and providing understandable text to better meet web user needs and contexts, as well as potential avenues for natural language processing for media accessibility.
```
Comment (iadawn) [inline _about/projects/wai-coop/symposium2.md:101]: Suggest chaning:

> 'amoung other'

to:

> 'for example'

**Comment by leticiaseixas** [comment]
Ready to merge

**Pull request merged**